WordPress/twentytwentyone
# Add padding to blocks in columns blocks

Fixes #270.

## Summary
Add padding to blocks in columns blocks.

## Relevant technical choices:
No.

## Test instructions

This PR can be tested by following these steps:

1. Create a column block with 3 columns.
2. Add both paragraphs and headings in the three columns.
3. When they don't have background images, these blocks should have a padding, both in the editor and in frontend.

## Quality assurance
